includes/wikia/models/tests/DesignSystemGlobalFooterModelTest.php

```diff
@@ -21,7 +21,7 @@ public function testGetLicensingAndVertical( $sitename, $rightsText, $rightsUrl,
 				[ 'wgSitename', $wikiId, $sitename ]
 			] ) );
 
-		$footerModel = new DesignSystemGlobalFooterModel( $wikiId );
+		$footerModel = new DesignSystemGlobalFooterModel( 'wikis', $wikiId );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wikis" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)